src/components/Header.jsx

```diff
@@ -0,0 +1,46 @@
+import logo from "../assets/images/logo.png";
+import logoMobile from "../assets/images/logo-mobile.png";
+import { Link, useNavigate } from "react-router-dom";
+import "./Header.css";
+
+const Header = ({ location }) => {
+  const nav = useNavigate();
+  const onClickButton = () => {
+    nav("/login");
+  };
+  return (
+    <>
+      <header className="Header">
+        <div className="header-left">
+          <Link to={"/"}>
```

Comment:
💊 제안
<Link to={"/"}>처럼 중괄호로 감싸지 않아도 되는 문자열 경로는 직접 작성하는 것이 더 간결하고 일반적인 컨벤션에 부합합니다. 

```suggestion
          <Link to="/">
```